packages/react/__tests__/__snapshots__/PublicAPI-test.js.snap

```diff
@@ -4820,127 +4820,10 @@ Map {
         "type": "bool",
       },
       "downshiftProps": Object {
-        "args": Array [
```

Comment:
@awarrier99 circling back, is there a way to retain the `downshiftProps` here? Looking at the Dropdown ts conversion, I do not see any snapshot changes to the `downshiftProps`. I'm trying to parse the difference in implementation to see what may be causing it 🤔 

Reference PR: https://github.com/carbon-design-system/carbon/pull/13022/files

Reply:
The other snapshot changes are intentional and are fine 👍🏻

Reply:
Sure, let me take a look at this

Reply:
@tw15egan this should be resolved now - I had changed the prop type to match what was in the `Dropdown` component originally, but I've now reverted that change